spec/registry.xml

```diff
@@ -0,0 +1,202 @@
+        instead of appending a minus sign. You can enable this formatting by setting the
+        currencySign option to "accounting".
+      -->
+      <option name="currencySign" values="accounting standard" default="standard"/>
+      <!-- The formatting that should be displayed for the number. -->
+      <option name="notation" values="standard scientific engineering compact" default="standard"/>
+      <!-- Numbering system to use. -->
+      <option name="numberingSystem" values="arab arabext bali beng deva fullwide gujr guru hanidec khmr knda laoo latn limb mlym mong mymr orya tamldec telu thai tibt"/>
+      <!-- When to display the sign for the number. -->
+      <!-- "negative" value is Experimental. -->
+      <option name="signDisplay" values="auto always exceptZero never" default="auto"/>
+      <!-- The formatting style to use. -->
+      <option name="style" values="decimal currency percent unit" default="decimal"/>
+      <!-- The unit to use in unit formatting.
+        Possible values are core unit identifiers, defined in UTS #35, Part 2, Section 6.
+        A subset of units from the full list was selected for use in ECMAScript.
+        Pairs of simple units can be concatenated with "-per-" to make a compound unit.
+        There is no default value; if the style is "unit", the unit property must be provided.
+      -->
+      <option name="unit" pattern="anythingNotEmpty"/>
+      <!-- The unit formatting style to use in unit formatting. -->
+      <option name="unitDisplay" values="long short narrow" default="short"/>
+      <!-- The minimum number of integer digits to use.
+        A value with a smaller number of integer digits than this number will be
+        left-padded with zeros (to the specified length) when formatted.
+      -->
+      <option name="minimumIntegerDigits" values="1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21" default="1"/>
+      <!-- The minimum number of fraction digits to use.
+        The default for plain number and percent formatting is 0;
+        the default for currency formatting is the number of minor unit digits provided by
+        the ISO 4217 currency code list (2 if the list doesn't provide that information).
+      -->
+      <option name="minimumFractionDigits" values="0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20"/>
+      <!-- The maximum number of fraction digits to use.
+        The default for plain number formatting is the larger of minimumFractionDigits and 3;
+        the default for currency formatting is the larger of minimumFractionDigits and the number of minor
+        unit digits provided by the ISO 4217 currency code list (2 if the list doesn't provide that information);
+        the default for percent formatting is the larger of minimumFractionDigits and 0.
+      -->
+      <option name="maximumFractionDigits" values="0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20"/>
```

Comment:
Of some relevance to us, TG2 is rather likely to very shortly update this and `minimumFractionDigits` to support values of up to 100 rather than 20: tc39/ecma402#786

Reply:
I've updated all of these instances to `positiveInteger`
We might even define regex for positive integers with / without zero, if we want to be more precise.